test-apps/presentation-test-app/config/rush-project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush-project.schema.json",
+  "operationSettings": [
+    {
+      "operationName": "build",
+      "outputFolderNames": ["lib"],
+      "disableBuildCacheForOperation": true
+    },
+    {
+      "operationName": "build:ci",
+      "outputFolderNames": ["lib"],
+      "disableBuildCacheForOperation": true
```

Comment:
It uses symlinks which are not supported by Rush's build cache.

Reply:
My original intent was to keep it and add a comment in the config file - but because the schema is linked I was yelled at for the `//` prefix. 

It probably makes sense to remove.